src/components/common/WorkCard/styles.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+
+import { colors } from '../../../utils'
+
+export const WorkWrapper = styled.div`
+  padding: 3.5rem;
+  background-color: ${colors.primary};
+  width: 100%;
+  height: 710;
+  color: ${colors.white};
+`
+
+export const TextWrapper = styled.div`
+  width: 90%;
+  margin: 2.5rem 4rem 0 1.6rem;
```

Comment:
weird margins with a 90% width..
I think you can do better
@jema28 the figma dimensions look so weird, can you tidy it up somehow?
https://www.figma.com/file/e7lI7XGEGvCAxKcnktUfkt/Yalla-Website-Redesign?node-id=534%3A946

Reply:
I just copied the styles on the Figma and that's how it turned out, it would be greater if @jema28 gave us specific dimensions that are like fixed

Reply:
you need to set box-sizing to border-box

Reply:
I think that's already done in the resetting file